docs/root/operations/cli.rst

```diff
@@ -79,7 +79,7 @@ following are the command line options that Envoy supports.
   *(optional)* The comma separated list of logging level per component. Non developers should generally 
   never set this option. For example, if you want `upstream` component to run at `debug` level and 
   `connection` component to run at `trace` level, you should pass ``upstream:debug,connection:trace`` to 
-  this flag.
+  this flag. See ``ALL_LOGGER_IDS`` in ``logger.h`` for a list of components.
```

Comment:
Do you mind making this a `:repo:` link? Search for other examples.

Reply:
will do, thanks!